fastlane/Fastfile

```diff
@@ -38,8 +38,10 @@ platform :ios do
     install_provisioning_profile(
       path: "ios-build.mobileprovision"
     )
-    disable_automatic_code_signing(
-      path: ENV["PROJECT_PATH"]
+    update_code_signing_settings(
+      use_automatic_signing: false,
+      path: ENV["PROJECT_PATH"],
+      targets: "Unity-iPhone"
```

Comment:
`ios-build-action` is not Unity-specific, so should not be hard-coded.
For example, I think should add new `Inputs` like this.
```
# README.md

### `disable-targets`

Targets to disable automatic code signing. Input targets separated by `,`. For example, `"MyApp,YourApp"`. Default `""`. (default to all targets)
```
And,
```ruby
# Fastfile

disable_targets = ENV["DISABLE_TARGETS"]&.split(/,/)
disable_automatic_code_signing(
  path: ENV["PROJECT_PATH"],
  targets: disable_targets
)
```
If you do this, unless user specify `disable-targets`, the behavior will be the same as before.